formatters.ts

```diff
@@ -0,0 +1,13 @@
+export function formatFatalError (message) {
```

Comment:
It does not have exported types but it has to be exported with the library otherwise the wrapper require will fail since it uses it to format fatal errors.